test/evp_test.c

```diff
@@ -134,6 +134,13 @@ static int pkey_check_fips_approved(EVP_PKEY_CTX *ctx, EVP_TEST *t)
      * value of approved.
      */
     int approved = 1;
+    const OSSL_PARAM *gettables = EVP_PKEY_CTX_gettable_params(ctx);
+
+    if (gettables == NULL
+        || OSSL_PARAM_locate_const(gettables,
+                                   OSSL_ALG_PARAM_FIPS_APPROVED_INDICATOR)
+                                   == NULL)
```

Comment:
style check says this line should align 8 chars in, rather than underneath the start of the arguments.  Not sure if you want to fix that or just waive the style

Reply:
moved it to the || which is still wrong according to the check :)